packages/kbn-eslint-plugin-css/src/rules/no_css_color.ts

```diff
@@ -0,0 +1,385 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { Rule } from 'eslint';
+import type { TSESTree } from '@typescript-eslint/typescript-estree';
+
+/**
+ * @description List of css properties that can that can apply color to html box element and text node
+ */
+const propertiesSupportingCssColor = ['color', 'background', 'backgroundColor', 'borderColor'];
```

Comment:
nit: technically, also `border` and `border-[side]-color`

Reply:
Switched this implementation, so now only specifying `border`,  `background` and `border` should suffice, let me know if I missed something